spec/factories/rating_votes.rb

```diff
@@ -4,7 +4,7 @@
     group { "experience_level" }
     rating { rand(1.0..8.0) }
     after(:build) do |vote|
-      vote.article ||= create(:article, user: vote.user)
+      vote.article ||= create(:article)
```

Comment:
for `Users::Delete` magical specs to work, the rating vote creator can't be the same as the article creator. This is because deleting a user will delete their articles and thus the rating votes attached to such article. So we shouldn't make a user vote on themselves in tests when it's already not permitted in the validation ;)